NEWS.md

```diff
@@ -550,6 +550,8 @@
 
 53. `as.data.frame(DT, row.names=)` no longer silently ignores `row.names`, [#5319](https://github.com/Rdatatable/data.table/issues/5319). Thanks to @dereckdemezquita for the fix and PR, and @ben-schwen for guidance.
 
+54. data.table construction uses `deparse` to try and automatically name un-named cols. The attempt is now limited to 1- line which gives substantial speedup in [some edge cases](https://github.com/Rdatatable/data.table/pull/5501). Thanks @OfekShilon for the report and fix, @michaelchirico for guidance.
+
```

Comment:
> data.table construction

Does that mean a `data.table(...)` call with arguments passed to it?  If so, could "data.table construction" be replaced with `data.table(...)` ?

> The attempt is now limited to 1- line

Does that mean calls to `data.table(...)` split over several lines (which I guess is very common) will lose this feature in the 2nd line onwards?  i.e.

```
data.table( A = 1:5,
            mean(my_vec)      # column name will be different now?
)
```

Reply:
I refined the phrasing a bit (see the [commit](https://github.com/Rdatatable/data.table/pull/5501/commits/d70ef652973c82b1192ae4d484f5e16470530ae7)) - it is now both more accurate and less understandable.. I'd be thankful for any suggestion, and am totally fine with omitting the NEWS item altogether. AFAICT this change has no user-facing consequences, beyond some speedup.

Reply:
Thanks. Ok I played with it a bit and tweaked the news item. It's not just 'some' speedup, but it scales with the data size. It's a tremendous finding certainly deserving of a news item and example!